docs/UserGuide.md

```diff
@@ -11,80 +11,82 @@
 
-1. Download the latest `letsgethired.jar` from [here](https://github.com/AY2324S1-CS2103T-W17-2/tp/releases).
+1. Download the **latest** `letsgethired.jar` from [our Github Releases](https://github.com/AY2324S1-CS2103T-W17-2/tp/releases).
 
     ![Zip file download](images/downloadJar.jpg)
 
 1. Copy the file to the folder you want to use as the _home folder_ for your **LetsGetHired** application.
 
-1. Open a command terminal, `cd` into the folder you put the jar file in, and use the `java -jar letsgethired.jar` command to run the application. A GUI similar to the below should appear in a few seconds. Note how the app contains some sample data.<br>
+1. Open a command terminal, `cd` into the folder you put the jar file in, and use the `java -jar letsgethired.jar` command to run the application. A GUI similar to the below should appear in a few seconds. Note that the app contains some sample data.<br>
    
     ![Ui](images/Ui.jpg)
 
-1. Here's what you'll see when you start the app for the first time ! We've added some intern applications already. Enter your commands in the **Command Bar**. On the left you'll see a **List View** of all your applications ordered by the date you added them. On the right, the **Select View** shows you a more in depth view of your selected application
+1. Here's what you'll see when you start the app for the first time! We've added some intern applications already. Enter your commands in the **Command Box** on the top of the app. On the bottom left you'll see a **List View** of all your applications ordered by the date you added them. On the right, the **Select View** shows you a more in depth view of your selected application.
 
-1. Type the command in the command box and press `Enter` to execute it. e.g. typing **`help`** and pressing Enter will open the help window. Some example commands you can try:
+1. Type the command in the Command Box and press `Enter` to execute it. e.g. typing **`help`** and pressing `Enter` will open the help window. Here are some example commands you can try:
 
     * `list` : Lists all internship applications.
 
-    * `add c/Summer r/Full Stack Developer n/John street` : Adds `Summer Internship - Full Stack Developer at John Street` application entry to the internship list
+    * `add n/Google c/Summer r/Full Stack Developer ` : Adds a new internship application entry to the internship list.
 
     * `delete 1` : Deletes the first internship application entry shown in the current list.
 
-    * `tag 1 s/Applied` : Updates the status of the 1st internship application entry shown in the current list as `Applied`.
+    * `edit 1 s/Accepted` : Updates the status of the first internship application entry shown in the current list as `Accepted`.
 
 1. Refer to the [Features](#features) below for details of each command.
+1. Follow the [Tutorial](#tutorial---managing-your-internship-applications) below to use LetsGetHired for your first internship application.
```

Comment:
Hmmm, I remember that it was recommended / better (safer) to use all the same numbers (e.g. 1.) since the md will still be interpreted as a numbered list. This prevents the case of numbers being skipped.